src/bundle.rs

```diff
@@ -257,9 +257,15 @@ fn export_template(lems: &LemsFile, nml: &[String], bundle: &str) -> Result<()>
         let doc = node.document().input_text();
         match node.tag_name().name() {
             "morphology" => {
-                let id = node
-                    .attribute("id")
-                    .ok_or_else(|| nml2_error!("Morph has no id"))?;
+                let cell = node
+                    .parent()
+                    .ok_or(nml2_error!("Morphology has no parent"))?;
```

Comment:
I _think_ that's incorrect. You can have this
```xml
<morphology id="m1">
  ...
</morphology>
<cell id="c1">
    <morphology id="m1" />
</cell>
<cell id="c0">
    <morphology id="m1" />
</cell>
```

Reply:
Thus you need to duplicate morphologies under `mrf`, allow references in `mrf`, or rename ids. Your choice ;)

Reply:
Horrible. Thanks for spotting... maybe default to a name like 'toplevel_m1' for these?

Reply:
I think the Arbor internal NML2 reader calls them _cell_ morphologies and just plain morphologies.